CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# Unreleased
+# 0.19.0 (2022-07-12)
+
+* Bump shred to version `0.14.1`, MSRV to 1.60.0 ([shred changelog][shred-changelog], [#756])
```

Comment:
i really would like to see some benchmarks on either specs or shred level to show that there is no performance regression to hashbrown. because all we did so far was assuming on some hash-algorithms that it will prob be fine.

But for us as veloren we prob won't update to this new versions if we cannot be sure that the performance has no regression in some benchmarks of the crate itself.

Reply:
> i really would like to see some benchmarks on either specs or shred level to show that there is no performance regression to hashbrown. because all we did so far was assuming on some hash-algorithms that it will prob be fine.

If you check the source code of `libstd`, you can see that `std::collections::HashMap` is just a wrapper around `hashbrown`:

```
use hashbrown::hash_map as base;

// ...

#[cfg_attr(not(test), rustc_diagnostic_item = "HashMap")]
#[stable(feature = "rust1", since = "1.0.0")]
#[rustc_insignificant_dtor]
pub struct HashMap<K, V, S = RandomState> {
    base: base::HashMap<K, V, S>,
}
```

I also checked the `rust-lang/rust` repo, it uses the latest version of `hashbrown`:

```rust
hashbrown = { version = "0.12", default-features = false, features = ['rustc-dep-of-std'] }
```

> [source](https://github.com/rust-lang/rust/blob/431c6f84990cdd08b68d46efff2efc75549934fe/library/std/Cargo.toml#L22)

I agree that we should add more benchmarks, but right now I don't have the capacity to do that.  However, since `hashbrown` is included in `std` anyways (just with a different default hasher) and we use the same hasher `hashbrown` uses by default, this change shouldn't affect performance at all, I just wanted to remove unnecessary dependencies.